source/MapDetailPanel.h

```diff
@@ -59,6 +61,10 @@ class MapDetailPanel : public MapPanel {
 	int governmentY = 0;
 	int tradeY = 0;
 
+	int planetNbr = 0;
```

Comment:
Please add some clarifying comments to those variables (and maybe also consider if the names are sufficiently descriptive)?